src/views/Settings/Header/index.vue

```diff
@@ -0,0 +1,18 @@
+<template lang="pug">
+  section
+    h1.fs-32.c-white Personalize graph
+    p.fs-16 Select currencies that you want to tran on monitor by default
+</template>
+
+<script>
+  export default {}
+</script>
+
+<style lang="sass" scoped>
+  h1
+    font-family: 'Roboto', sans-serif
```

Comment:
Stylowaniem się póki co za bardzo nie przejmujemy, ale warto w takich przypadkach używać tego co już jest w projekcie, czyli tutaj np. `/styles/vars/_fonts.scss`, gdzie mamy już stałą `$ff-roboto` (tak samo z kolorem niżej -> `/styles/vars/_colors.scss`). Importujemy to np. `@import "~styles/vars/colors"`